docs/get-started/migrate-from-docker-compose.md

```diff
@@ -0,0 +1,494 @@
+|----------------|-------------|-------|
+| `volumes: ["data:/app/data"]` | `.WithVolume("data", "/app/data")` | Named volume |
+| `volumes: ["./host:/container"]` | `.WithBindMount("./host", "/container")` | Bind mount |
+| `volumes: ["./config:/app:ro"]` | `.WithBindMount("./config", "/app", isReadOnly: true)` | Read-only bind mount |
+
+### Dependencies and ordering
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `depends_on: [db]` | `.WithReference(db)` | Service dependency with connection |
+| `depends_on: db: condition: service_started` | `.WaitFor(db)` | Wait for service start |
+| `depends_on: db: condition: service_healthy` | `.WaitForCompletion(db)` | Wait for health check |
+
+### Networks
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `networks: [backend]` | Automatic | .NET Aspire handles networking automatically |
+| Custom networks | Not needed | Service discovery handles inter-service communication |
+
+### Resource limits
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `deploy.resources.limits.memory: 512m` | `.WithResourceLimits(memory: 0.5)` | Memory limit in GB |
+| `deploy.resources.limits.cpus: 0.5` | `.WithResourceLimits(cpu: 0.5)` | CPU limit |
+
+### Health checks
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `healthcheck.test: ["CMD", "curl", "http://localhost/health"]` | Built-in for integrations | .NET Aspire integrations include health checks |
+| `healthcheck.interval: 30s` | Configurable in integration | Health check configuration varies by resource type |
+
+### Restart policies
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `restart: unless-stopped` | `.WithRestart(RestartPolicy.UnlessStopped)` | Available in container resources |
+| `restart: always` | `.WithRestart(RestartPolicy.Always)` | Always restart |
+| `restart: no` | Default | No restart policy |
+
+### Labels and metadata
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `labels: app=myapp` | `.WithAnnotation("app", "myapp")` | Resource annotations |
+
+### Logging
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `logging.driver: json-file` | Built-in | .NET Aspire provides integrated logging |
+| `logging.options.max-size: 10m` | Dashboard configuration | Managed through .NET Aspire dashboard |
+
+### Database services
+
+| Docker Compose | .NET Aspire | Notes |
+|----------------|-------------|-------|
+| `image: postgres:15` | `builder.AddPostgres("name")` | PostgreSQL with automatic configuration |
+| `image: mysql:8` | `builder.AddMySql("name")` | MySQL with automatic configuration |
+| `image: redis:7` | `builder.AddRedis("name")` | Redis with automatic configuration |
+| `image: mongo:latest` | `builder.AddMongoDB("name")` | MongoDB with automatic configuration |
```

Comment:
I think this reference section could perhaps be its own article (`ms.topic: reference` frontmatter and all), and maybe the TOC could have a `Migrate to Aspire` node with two children, this article named `From Docker Compose` and the new reference `Docker Compose to AppHost API Reference`. Otherwise, maybe just push this content to the bottom of this article.

Reply:
Yes, I would having looked at it again. Please do so as I've described.